libhoney.go

```diff
@@ -235,6 +261,7 @@ type dynamicField struct {
 //
 // Make sure to call Close() to flush buffers.
 func Init(config Config) error {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Ha wow I did not know that either. I'm a little wary of seeding the global RNG, since it would be quite the head-scratcher if calling `libhoney.Init` broke unrelated tests or something that expect a deterministic `rand`). But ¯\_(ツ)_/¯